ohdear/resource_site_test.go

```diff
@@ -16,7 +16,31 @@ func TestAccOhdearSiteCreate(t *testing.T) {
 			{
 				Config: testConfigForOhdearSiteCreate(ri),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(fmt.Sprintf("ohdear_site.test-%d", ri), "team_id", "1775"),
+					resource.TestCheckResourceAttr(fmt.Sprintf("ohdear_site.test-%d", ri), "team_id", "1820"),
+					resource.TestCheckResourceAttr(fmt.Sprintf("ohdear_site.test-%d", ri), "url", fmt.Sprintf("http://www.test-%d.com", ri)),
+				),
+			},
+		},
+	})
+}
+
+func TestAccOhdearSiteLifecycle(t *testing.T) {
+	ri := acctest.RandInt()
+	resource.Test(t, resource.TestCase{
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testConfigForOhdearSiteCreate(ri),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
Can we check the API for the existence of these? Just to make sure it is getting created on remote. I think it is a testing best practice. Or at least I see most of the providers doing it. https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_cloudfront_public_key_test.go#L26

Reply:
Maybe you and I can pair on this - I think its a good exercise for gleaning some more data from you on idiomatic go.

Reply:
Yea for sure, let's do it